test/intersections.jl

```diff
@@ -17,6 +17,10 @@
     @test s1 ∩ s2 == Segment(P2(0.5,0.0), P2(1,0))
     @test s2 ∩ s1 == Segment(P2(0.5,0.0), P2(1,0))
 
+    s1 = Segment(P2(0,0), P2(1,-1))
+    s2 = Segment(P2(0.5,-0.5), P2(1.5,-1.5))
+    @test s1 ∩ s2 == s2 ∩ s1 == Segment(P2(0.5,-0.5), P2(1,-1))
```

Comment:
Aren't s1 and s2 parallel without intersection? Shouldn't the return be NoIntersection? Sorry if I misread the coordinates somehow, just plotted it on my head.

Reply:
![grafik](https://user-images.githubusercontent.com/67096719/196358080-4ae6adb0-249c-4f21-ab70-efaea9c46f77.png)

they are collinear with negative slope, that caused the old version of intersectcollinear to be erroneous.